src/pipecat/services/asyncai/tts.py

```diff
@@ -0,0 +1,513 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""Async text-to-speech service implementations."""
+
+import base64
+import json
+import uuid
+from typing import AsyncGenerator, Optional
+
+from loguru import logger
+from pydantic import BaseModel
+import asyncio
+import aiohttp
+
+from pipecat.frames.frames import (
+    CancelFrame,
+    EndFrame,
+    ErrorFrame,
+    Frame,
+    StartFrame,
+    StartInterruptionFrame,
+    TTSAudioRawFrame,
+    TTSStartedFrame,
+    TTSStoppedFrame,
+)
+from pipecat.processors.frame_processor import FrameDirection
+from pipecat.services.tts_service import AudioContextWordTTSService, TTSService
+from pipecat.transcriptions.language import Language
+from pipecat.utils.asyncio.watchdog_async_iterator import WatchdogAsyncIterator
+from pipecat.utils.tracing.service_decorators import traced_tts
+
+try:
+    import websockets
```

Comment:
Add these imports for the websockets changes I'm suggesting. We're importing `connect` as `websocket_connect` to help distinguish from other uses of "connect".
```suggestion
    import websockets
    from websockets.asyncio.client import connect as websocket_connect
    from websockets.protocol import State
```